src/index.html

```diff
@@ -4,9 +4,362 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap"    rel="stylesheet">
+
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <div class="header__image">
+        <a href="#" class="header__logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
```

Comment:
will be better to specify anchor links instead of using single # symbol
```suggestion
            <a href="#apple" class="nav__link is-active">
```